runtime/ast/statement.go

```diff
@@ -251,6 +331,36 @@ func (s *ForStatement) EndPosition() Position {
 	return s.Block.EndPosition()
 }
 
+const forStatementForKeywordSpaceDoc = prettier.Text("for ")
+const forStatementSpaceInKeywordSpaceDoc = prettier.Text(" in ")
+
+func (s *ForStatement) Doc() prettier.Doc {
+	doc := prettier.Concat{
```

Comment:
need to concat?

Reply:
Yes, we always need the concatenation, as https://github.com/onflow/cadence/pull/1280/files#diff-cb6a372e592ff206bc8f5ad13e85ba6527d1e6914f281f0dd263479c7f3a278eR350 always appends more after the conditionally appended index (https://github.com/onflow/cadence/pull/1280/files#diff-cb6a372e592ff206bc8f5ad13e85ba6527d1e6914f281f0dd263479c7f3a278eR342)